odoo/fields.py

```diff
@@ -660,6 +660,37 @@ def get(key, get_context=env.context.get):
 
         return tuple(get(key) for key in self.depends_context)
 
+    def mapped(self, records):
```

Comment:
docstring

Reply:
Why does `field.mapped()` return a list of cache-formatted values?
I would expect `field.mapped()` to be equivalent to the multi version of `field.__get__()`.